source/Core/Inc/Settings.h

```diff
@@ -49,32 +49,33 @@ enum SettingsOptions {
   TempChangeLongStep             = 26, // Temperature-change-increment on long button press
   TempChangeShortStep            = 27, // Temperature-change-increment on short button press
   HallEffectSensitivity          = 28, // Operating mode of the hall effect sensor
-  AccelMissingWarningCounter     = 29, // Counter of how many times we have warned we cannot detect the accelerometer
-  PDMissingWarningCounter        = 30, // Counter of how many times we have warned we cannot detect the pd interface
-  UILanguage                     = 31, // Selected UI Language code, null-terminated *only if* the length is less than 8 chars
-  PDNegTimeout                   = 32, // PD timeout in 100ms steps
-  OLEDInversion                  = 33, // Invert the colours on the display
-  OLEDBrightness                 = 34, // Brightness for the OLED display
-  LOGOTime                       = 35, // Duration the logo will be displayed for
-  CalibrateCJC                   = 36, // Toggle calibrate CJC at next boot
-  BluetoothLE                    = 37, // Toggle BLE if present
-  USBPDMode                      = 38, // Toggle PPS & EPR
-  ProfilePhases                  = 39, // Number of profile mode phases
-  ProfilePreheatTemp             = 40, // Temperature to preheat to before the first phase
-  ProfilePreheatSpeed            = 41, // Maximum allowed preheat speed in degrees per second
-  ProfilePhase1Temp              = 42, // Temperature to target for the end of phase 1
-  ProfilePhase1Duration          = 43, // Target duration for phase 1
-  ProfilePhase2Temp              = 44, // Temperature to target for the end of phase 2
-  ProfilePhase2Duration          = 45, // Target duration for phase 2
-  ProfilePhase3Temp              = 46, // Temperature to target for the end of phase 3
-  ProfilePhase3Duration          = 47, // Target duration for phase 3
-  ProfilePhase4Temp              = 48, // Temperature to target for the end of phase 4
-  ProfilePhase4Duration          = 49, // Target duration for phase 4
-  ProfilePhase5Temp              = 50, // Temperature to target for the end of phase 5
-  ProfilePhase5Duration          = 51, // Target duration for phase 5
-  ProfileCooldownSpeed           = 52, // Maximum allowed cooldown speed in degrees per second
+  HallEffectSleepTime            = 29, // Seconds timeout to sleep when hall effect over threshold
```

Comment:
Yeah this has to go at the end, these index's are used for setting migration between reflash so new settings need to be appended :(

Reply:
done in https://github.com/Ralim/IronOS/pull/1969/commits/fdefb5bc727c473918c7eaba23d3959e288545a9 + https://github.com/Ralim/IronOS/pull/1969/commits/fb904a532b43c18c6ff56ab070a26784d6db5a94